setup_cfg_fmt.py

```diff
@@ -196,6 +196,51 @@ def _python_requires(
         return _format_python_requires(minimum, excluded)
 
 
+def _install_requires(cfg: configparser.ConfigParser) -> List[str]:
+    raw = cfg.get('options', 'install_requires', fallback='')
+
+    install_requires = raw.strip().splitlines()
+
+    normalized = sorted(
+        (_normalize_req(req) for req in install_requires),
+        key=lambda req: (';' in req, req),
+    )
+
+    if len(normalized) > 1:
+        return ['', *normalized]
```

Comment:
`req.insert(0, '')` might be simpler / faster

Reply:
This way doesn't mutate the object but I can make the change

Reply:
this function created / owns the object so nbd